contracts/contracts/abstract/SelfVerificationRoot.sol

```diff
@@ -69,27 +69,53 @@ abstract contract SelfVerificationRoot is ISelfVerificationRoot {
     /// @dev Triggered in verifySelfProof when attestation ID validation fails
     error InvalidAttestationId();
 
+    // ====================================================
+    // Events
+    // ====================================================
+
+    /// @notice Emitted when the verification configuration is updated
+    event VerificationConfigUpdated(ISelfVerificationRoot.VerificationConfig indexed verificationConfig);
+
+    /// @notice Emitted when the verification is successful
+    event VerificationSuccess(uint256[3] revealedDataPacked, uint256 indexed userIdentifier, uint256 indexed nullifier);
+
+    /// @notice Emitted when the scope is updated
+    event ScopeUpdated(uint256 indexed newScope);
+
+    /// @notice Emitted when a new attestation ID is added
+    event AttestationIdAdded(uint256 indexed attestationId);
+
+    /// @notice Emitted when an attestation ID is removed
+    event AttestationIdRemoved(uint256 indexed attestationId);
+
     /**
      * @notice Initializes the SelfVerificationRoot contract.
-     * @param identityVerificationHub The address of the Identity Verification Hub.
-     * @param scope The expected proof scope for user registration.
-     * @param attestationIds The expected attestation identifiers required in proofs.
+     * @param _identityVerificationHubAddress The address of the Identity Verification Hub.
+     * @param _scopeValue The expected proof scope for user registration.
+     * @param _attestationIds The expected attestation identifiers required in proofs.
      */
-    constructor(address identityVerificationHub, uint256 scope, uint256[] memory attestationIds) {
-        _identityVerificationHub = IIdentityVerificationHubV1(identityVerificationHub);
-        _scope = scope;
-        for (uint256 i = 0; i < attestationIds.length; i++) {
-            _attestationIds[attestationIds[i]] = true;
+    constructor(address _identityVerificationHubAddress, uint256 _scopeValue, uint256[] memory _attestationIds) {
```

Comment:
For the arguments declared in our contract, I just followed this pattern
- Function Argument Names
https://docs.soliditylang.org/en/latest/style-guide.html#function-argument-names
- Avoiding Naming Collisions
https://docs.soliditylang.org/en/latest/style-guide.html#avoiding-naming-collisions
- Underscore Prefix for Non-external Functions and Variables
https://docs.soliditylang.org/en/latest/style-guide.html#underscore-prefix-for-non-external-functions-and-variables

It seems not all contracts in production follows this rules. Actually, the rules are so much variety, but at least, we should follow this rule.